### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by tapnikita** [Droppers/AnimatedBottomBar#45]
You have 2 Options for Text and Icon, What can you add 2 more:

Text and Text
Icon and Icon

So people can choose if I just want icon.

Also maybe add a new variation,
Icon with Text On Top or Bottom.

Options can be TextIcon

app:abb_selectedTabType="text/icon/both/icononly/textonly"
app:abb_TextLocation="top/bottom"

**Comment by irvine752** [comment]
This is a great library with outstanding flexibility, but I have agree with @tapnikita this feature is needed. Depending on the use case, sometimes it's best to give the user more options. 

Maybe it could be as easy as disabling the animation from switching text to icon or icon to text?

**Comment by Droppers** [comment]
I'm not sure what this would look like. Does this mean no animation for the text or icon? If so, how would this library be different from a conventional bottom bar.

**Comment by irvine752** [comment]
Yep, no animation for text or icon which would make it similar to the conventional bottom bar. The animation can always be turned on or off depending on the user's preference. The idea is to give the user more flexibility.

**Comment by tapnikita** [comment]
> I'm not sure what this would look like. Does this mean no animation for the text or icon? If so, how would this library be different from a conventional bottom bar.

I do Hope you add the feature just to have text and text and icon and icon, Animation control could be also nice. Hope this gets added.
